lesson5/src/main/java/com/andersenlab/util/InvalidTicketsCounter.java

```diff
@@ -0,0 +1,19 @@
+package com.andersenlab.util;
+
+import com.andersenlab.entity.BusTicket;
+
+import java.lang.reflect.Method;
+import java.util.List;
+
+public class InvalidTicketsCounter {
+    public static int countInvalidTickets(Validator obj, String methodName, List<BusTicket> inputList) {
+        try {
+            Method method = obj.getClass().getMethod(methodName, List.class);
```

Comment:
it is not the best idea to use reflection where it doesn't really needed. This could crate performance issues